adding_data_long.qmd

```diff
@@ -1252,11 +1253,11 @@ Then in R
 ```{r, eval=FALSE}
 dataset_ids <- "Smith_1995"
 
-# Update remake file
-build_setup_pipeline()
+# Update build file
+build_setup_pipeline(method="base")
 
 # check data builds
-remake::make(dataset_ids)
+build(dataset_ids)
```

Comment:
That won't work, aster's no function called build. You'd need to run the specific lines of code from the `build.R` file